### PR TITLE
Expand coverage for cipher wrapping to more issue IDs

### PR DESCRIPTION
**Pull request by rachkor**
### Summary
We have logic built in to the plugin that wraps ciphers in code blocks if the Issue ID matches a list of Issues known to have ciphers in them. Based on a sample file a team sent us, this PR expands the coverage to more IDs that are known to contain ciphers.

### CODE REVIEW

File: lib/nexpose/vulnerability.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def supported_tags
         :references, :tags,
 
         # evidence tag
-        :details
+        :details, :name
```
Comment (etdsoft) [inline lib/nexpose/vulnerability.rb]: what's this #name here? It seems unrelated to the PR?

Comment (rachkor) [reply lib/nexpose/vulnerability.rb]: Thanks for catching this! Looks like this was due to my lack of git skillz as it wasn't supposed to be in here. I just pushed a commit to fix it.
